cpp/include/cudf/rolling/range_window_bounds.hpp

```diff
@@ -38,15 +38,15 @@ struct range_window_bounds {
   /**
    * @brief Factory method to construct a bounded window boundary.
    *
-   * @param value Finite window boundary
+   * @param scalar_ Finite window boundary
    *
    */
-  static range_window_bounds get(scalar const&);
+  static range_window_bounds get(scalar const& scalar_);
```

Comment:
How about `boundary` instead of `scalar_` ?

Reply:
updated